packages/next/src/build/index.ts

```diff
@@ -293,6 +293,7 @@ export default async function build(
 
       const { headers, rewrites, redirects } = customRoutes
       NextBuildContext.rewrites = rewrites
+      NextBuildContext.redirects = redirects
```

Comment:
We might want to use the `_originalRedirects` value since with `i18n` the `source` values are transformed to include dynamic segments which would break them from matching the types x-ref: https://github.com/vercel/next.js/blob/16d7ffdd11f04f459194f19c55d3d452e38021b9/packages/next/src/lib/load-custom-routes.ts#L572

Reply:
Nvm that's still needed.